modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-sets/actions/getFieldsWithoutOptions.es.js

```diff
@@ -18,28 +18,29 @@ export default function getFieldsWithoutOptions(
 	dataDefinitionFields,
 	defaultLanguageId
 ) {
-	const fieldTypesWithOptions = [
-		FIELD_TYPES.grid,
-		FIELD_TYPES.radio,
-		FIELD_TYPES.checkboxMultiple,
-		FIELD_TYPES.select,
-	];
+	const hasValidOption = (options) =>
+		options.some(({label, value}) => label && value);
 
-	const fieldHasOptions = (options) =>
-		options[defaultLanguageId][0].edited &&
-		options[defaultLanguageId][0].label;
-
-	return dataDefinitionFields.filter((field) => {
-		const {customProperties, fieldType} = field;
-
-		return (
-			fieldTypesWithOptions.includes(fieldType) &&
-			(customProperties.options
-				? !fieldHasOptions(customProperties.options)
-				: !(
-						fieldHasOptions(customProperties.columns) &&
-						fieldHasOptions(customProperties.rows)
-				  ))
-		);
+	return dataDefinitionFields.filter(({customProperties, fieldType}) => {
+		switch (fieldType) {
+			case FIELD_TYPES.radio:
+			case FIELD_TYPES.checkboxMultiple:
```

Comment:
Could we remove these case, since are not using?

Reply:
Actually these cases are behaving the same as the `case FIELD_TYPES.select`

[more info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch)